File: Easy-problems/hidden-digits.cs
using System;
using System.IO;
using System.Linq;
using System.Text;


class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        {
            while (!reader.EndOfStream)
            {
                string line = reader.ReadLine();
                if (String.IsNullOrEmpty(line))
                    continue;
                // do something with line
                char[] dictLetters = { 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j' };
                char[] dictNumbers = { '0', '1', '2', '3', '4', '5', '6', '7', '8', '9' };
                StringBuilder result = new StringBuilder();
                foreach (char letter in line)
                {
                    if (Array.IndexOf(dictLetters, letter) > -1)
                    {
                        result.Append(Array.IndexOf(dictLetters, letter));
                    }
                    else if (Array.IndexOf(dictNumbers, letter) > -1)
                    {
                        result.Append(letter);
                    }
                }

                if (result.Length == 0)
                {
                    result.Append("NONE");
                }

                Console.WriteLine(result);
                
            }

        }

    }
}

File: Easy-problems/hidden-digits.java
import java.io.*;
import java.util.*;


public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            // Process line of input Here


            String dictLetters = "abcdefghij";
            String dictNumbers = "0123456789";
            StringBuilder result = new StringBuilder();
            for (char letter : line.toCharArray())
            {
                if (dictLetters.indexOf(letter) > -1)
                {
                    result.append(dictLetters.indexOf(letter));
                }
                else if (dictNumbers.indexOf(letter) > -1)
                {
                    result.append(letter);
                }
            }

            if (result.length() == 0)
            {
                result.append("NONE");
            }

            System.out.println(result);
            
        }
        
        buffer.close();
    }
    
    
    
}
